neo4j-cypher-dsl/src/test/java/org/neo4j/cypherdsl/core/CypherIT.java

```diff
@@ -3951,10 +3956,17 @@ void prettyPrintingWithDefaultSettingShouldWork() {
 				.isEqualTo(
 					"MATCH (u:User)\n" +
 					"WHERE (u.name = 'Max'\n" +
-					"  AND u.lastName = 'Mustermann')\n" +
+					"  AND u.lastName = 'Mustermann'\n" +
+					"  AND EXISTS {\n" +
+					"    MATCH (u)-[:LIKES]->(b:Bike)\n" +
+					"    WHERE ((b)-[:LINK]->(other)\n" +
+					"      OR EXISTS {\n" +
+					"        MATCH (b)-[:LINK]->(other)\n" +
+					"      })\n" +
+					"  })\n" +
 					"SET u.lastName = $newName\n" +
 					"WITH u\n" +
-					"MATCH (b:Bike)\n" +
+					"MATCH (b)\n" +
```

Comment:
@michael-simons is it a bug or a feature that the Label is not printed here? I guess I have to use a different node-instance for the subquery, isn't it?

Reply:
This is a bug. `b` would here not match the bike but any node. I'll fix that.